third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -738,11 +738,17 @@ def arcs_kt_gen(
     schema_name = name + "_schema"
     plan_name = name + "_plan"
 
+    manifest_deps = [
+        d + "_manifest"
+        for d in manifest_only(deps, inverse = True)
+        if (d.find(":") != -1 and not d.endswith("_manifest"))
```

Comment:
`d.find(":") != -1` could be written more simply: `d[0] != ":"`

Reply:
It won't always be the first symbol. 

I originally had `d.startswith(":")`, but it failed in certain cases.

Reply:
Ah right, I misunderstood the logic for `find`

This is now my new suggestion: `if ":" in d and not ...`

Reply:
I'm happy with that suggestion, I'll apply the change tomorrow :)